ci/build-docker-image.sh

```diff
@@ -63,7 +63,15 @@ case $image_substring in
 esac
 
 # Create tag based on branch name
-CI_DOCKER_IMAGE_NAME="$CI_DOCKER_IMAGE_NAME":$(echo "$CI_BRANCH" | tr '[:upper:]' '[:lower:]' | sed -e 's:/:_:g')
+default_branch=$(git remote show origin | grep HEAD | awk '{print $3;}')
```

Comment:
Is the `;` need after the `$3`? As we don't have it in our `__tue-git-default-branch` function

Reply:
It is an optional syntax if there is only one command being executed. If multiple commands are executed then they must be separated by semi colons